config.yml

```diff
@@ -10,6 +10,7 @@ sites:
       - pmc-deadline-2019.wpcom.test
       - pmc-footwearnews-2018.wpcom.test
       - pmc-hollywoodlife-2017.wpcom.test
+      - pmc-sportico-2020.wpcom.test
```

Comment:
Sportico shouldn't be included here because it isn't a VIP Classic site.

Reply:
removed!